chart/templates/mayastor/io/io-engine-daemonset.yaml

```
@@ -98,7 +98,8 @@ spec:
         - "-T={{ .Values.io_engine.target.nvmf.iface }}"{{ end }}{{ if .Values.io_engine.envcontext }}
         - "--env-context=--{{ .Values.io_engine.envcontext }}"{{ end }}{{ if .Values.io_engine.reactorFreezeDetection.enabled }}
         - "--reactor-freeze-detection"{{ end }}
-        - "--tgt-crdt={{ .Values.io_engine.target.nvmf.hostCmdRetryDelay.crdt1 }}"
+        - "--tgt-crdt={{ .Values.io_engine.target.nvmf.hostCmdRetryDelay.crdt1 }}"{{ if .Values.eventing.enabled }}
+        - "--events-url=nats://{{ .Release.Name }}-nats:4222"{{ end }}
```

Whys is this added ?

---

It is needed to enable eventing in io-engine and pass mbus-url

---

Should the eventing flags be inside an if like the ones on the eventing components? Not just this one, all of the others as well?
```suggestion
{{- if .Values.eventing.enabled }}
        - "--events-url=nats://{{ .Release.Name }}-nats:4222"{{ end }}
{{- end }}        
```

---

Only when the "events-url" is passed, the eventing is enabled in the core agent or io-engine. So I am passing "events-url" when eventing is enabled.

---

Oh, okay the if's are over to the right. Sorry, I didn't notice it. Great!